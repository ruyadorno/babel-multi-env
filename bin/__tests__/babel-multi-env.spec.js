import path from "path";
import { execSync } from "child_process";

describe("babel-multi-env bin", () => {
  it("should compile runnable script on the platform", () => {
    execSync(
      "node bin/babel-multi-env.js --multi-versions 8.0.0 6.0.0 4.0.0 0.12.0 0.10.0 --given bin/__fixtures__/arrayJoin.fixture.js --out-dir tmp"
    );
    const result = execSync("node tmp/arrayJoin.fixture.js");
    expect(result.toString("utf8")).toMatchSnapshot();
  });
});
