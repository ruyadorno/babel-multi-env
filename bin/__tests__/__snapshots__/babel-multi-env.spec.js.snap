// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel-multi-env bin should compile runnable script on the platform 1`] = `
"This is a pen 
"
`;
